src/Workspaces/VisualBasic/Portable/AddImports/VisualBasicAddImportsService.vb

```diff
@@ -0,0 +1,48 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.AddImports
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.AddImports
+    Friend Class VisualBasicAddImportsService
+        Inherits AbstractAddImportsService(Of
+            CompilationUnitSyntax,
+            NamespaceBlockSyntax,
+            ImportsStatementSyntax,
+            ImportsStatementSyntax)
+
+        Protected Overrides Function GetAlias(usingOrAlias As ImportsStatementSyntax) As SyntaxNode
+            Return usingOrAlias.ImportsClauses.OfType(Of SimpleImportsClauseSyntax).
+                                               FirstOrDefault()?.Alias
```

Comment:
What about when there's multiple ImportClauses and a later one is an Alias? Or multiple? (I don't know if those are valid cases in VB, but the FirstOrDefault() feels sketchy)